docs/01-basics/00-basics.adoc

```diff
@@ -14,11 +14,12 @@
 
 // end::EN[]
 
+include::../what-to-expect-of-this-module.adoc[{include_configuration}]
```

Comment:
> Including from a parent folder feels.. wrong.
I had the same feeling... yet, it was not about this file but with the images for the timing pie chart.

Reply:
I agree. Go for it 🙂

Reply:
Images have to be put into the "images" folder, because that is defined as source-folder for all images.
I created a new folder called `01-basics` in the global images folder, where these images now reside in. The adoc file was moved back to the basics folder.